internal/cli/arguments.go

```diff
@@ -31,6 +34,42 @@ func (a *Argument) Ask(cmd *cobra.Command, value interface{}) error {
 	return askArgument(cmd, a, value)
 }
 
+type pickerOptionsFunc func() (pickerOptions, error)
+
+func (a *Argument) Picker(cmd *cobra.Command, result *string, fn pickerOptionsFunc) error {
```

Comment:
Given that the other methods of args and flags are verbs, can we change this to be `Pick`?